pgsmo/objects/table/templates/+default/nodes.sql

```diff
@@ -8,12 +8,14 @@
 SELECT  rel.oid,
         (SELECT count(*) FROM pg_trigger WHERE tgrelid=rel.oid) AS triggercount,
         (SELECT count(*) FROM pg_trigger WHERE tgrelid=rel.oid AND tgenabled = 'O') AS has_enable_triggers,
+        (CASE WHEN rel.relkind = 'p' THEN true ELSE false END) AS is_partitioned,
         nsp.nspname AS schema,
         nsp.oid AS schemaoid,
         rel.relname AS name,
         {{ SYSOBJECTS.IS_SYSTEMSCHEMA('nsp') }} as is_system
 FROM pg_class rel
 INNER JOIN pg_namespace nsp ON rel.relnamespace= nsp.oid
-    WHERE rel.relkind IN ('r','t','f') 
+    WHERE rel.relkind IN ('r','t','f','p')
```

Comment:
I believe this file and `9.1_plus/nodes.sql` are the only changes necessary to perform the fix.

It may be the wrong decision, but I also added `'p'` to allowed [relkind](https://www.postgresql.org/docs/current/catalog-pg-class.html) values in other queries. My reasoning is that if we're getting e.g. ACL then we should get it for partitioned tables too. Hope this doesn't break other functionality.